README.md

```diff
@@ -119,14 +119,15 @@ Example:
  curl -vvv "127.0.0.1:8080/api/codex/v1/download/<Cid of the content>" --output <name of output file>
  ```
 
-### `/api/codex/v1/upload`
+### `/api/codex/v1/upload[?chunk=N]`
 
 Upload a file, upon success returns the `Cid` of the uploaded file.
+The optional 'chunk' parameter specifies Manifest.blockSize.
```

Comment:
I would not mention the `Manifest`, I would just say that chunks/blocks are going to of the specified size and padded up to `N`.

Reply:
Padding is our internal machinery. User of REST API don't see it since download now returns non-padded file

Reply:
`Manifest.blockSize` is also internal machinery and there is no way as of now to get a manifest from codex as of now, tho that might change. 

I would keep `Manifest.blockSize` out just say that chunks will be of a maximum size of `N`.